addons/l10n_in_edi_ewaybill/views/account_move_views.xml

```diff
@@ -49,4 +49,14 @@
             </xpath>
         </field>
     </record>
- </odoo>
+    <record id="invoice_form_inherit_l10n_in_edi_ewaybill_1" model="ir.ui.view">
+        <field name="name">account.move.form.inherit.l10n.in.ewaybill_1</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account_edi.view_move_form_inherit"/>
+        <field name="arch" type="xml">
+            <xpath expr="//div[hasclass('alert', 'alert-info')]" position="attributes">
+                <attribute name="class">alert alert-info o_row</attribute>
```

Comment:
What do you want to achieve with this?  We can not just change it in account module itself?  (attributes overriding complicates inheritance)  Maybe the xpath can be more specific?

Otherwise, without the cancel wizard, we can go forward.  You are sure you want to do all the fw-ports from 16.0?

Reply:
> What do you want to achieve with this? We can not just change it in account module itself? (attributes overriding complicates inheritance) Maybe the xpath can be more specific?

Hello @jco-odoo,
Previously, the "Process Now" button was displayed on a separate line. 
![ewaybill process now](https://github.com/odoo/odoo/assets/124334471/20e27511-424f-4294-baad-1b3ec2d4444f)

With the adjustment made, it's now aligned with the content on the same line, improving the layout.
![ewaybill process now after](https://github.com/odoo/odoo/assets/124334471/5fa784c4-408b-4aed-8e79-ccdf4185195d)

Reply:
> Either you change this in the account_edi module itself or we just skip this change and do this in master, because why bother to still change this in 16? @ajjo-odoo

yes sure, it's fine to have this in master